src/main/resources/avro/referencemethods.avdl

```diff
@@ -284,6 +355,12 @@ record GetSequenceBasesRequest {
   union { null, long } end = null;
 
   /**
+  Specifies the maximum number of bases to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
```

Comment:
I'm not sure what the best thing to do is here --- we should tackle it in a separate PR though, and try to get this one merged as soon as we can.